plugin.php

```diff
@@ -29,6 +29,12 @@
 
 namespace HumanMade\SimpleSaml;
 
+// is_plugin_active_for_network can only be used once the plugin.php file is
+// included.
+if ( ! function_exists( 'is_plugin_active_for_network' ) ) {
+	require_once( ABSPATH . '/wp-admin/includes/plugin.php' );
+}
```

Comment:
This should be inside `bootstrap()` instead IMO

Reply:
Aww good catch! I went ahead and made that fix & re-tested. Let me know if there are any issues. 🚀